src/Microsoft.TestPlatform.Extensions.TrxLogger/Microsoft.TestPlatform.Extensions.TrxLogger.csproj

```diff
@@ -40,11 +40,7 @@
     <ProjectReference Include="$(RepoRoot)src\Microsoft.TestPlatform.CoreUtilities\Microsoft.TestPlatform.CoreUtilities.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
-    <PackageReference Include="System.Security.Principal.Windows" Version="4.0.0" />
-  </ItemGroup>
-
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
```

Comment:
Not seeing how this condition would ever be true given `<TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>`.   Why can't 5.0 be referenced for all non `NetFrameworkMinimum` scenarios?  This change is needed because there is a package downgrade warning with upgrading Microsoft.Win32.Registry in https://github.com/microsoft/vstest/pull/5191/files#diff-cb0ebd35c90417a3701f998bc4ba8f981bb5fd0c4c0b6f078c722bb73f5f5434R20

Reply:
On anything newer than .NET 5, this package shouldn't be referenced anymore as the library got moved inbox.

> Not seeing how this condition would ever be true given <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>

This logic adds the latest and previous .NETCoreApp TFMs to all projects:

https://github.com/microsoft/vstest/blob/ffe9e9aec40de57ede1ddc7e1b23a47032010c30/Directory.Build.targets#L10-L24